tests/test_policy.py

```diff
@@ -500,6 +500,66 @@ def test_resource_type_empty_metadata(self):
         if empty:
             raise ValueError("Empty Resource Metadata %s" % (', '.join(empty)))
 
+    def test_valid_arn_type(self):
+        arn_db = load_data('arn-types.json')
+        invalid = {}
+        overrides = {'wafv2': set(('webacl',))}
+
+        # we have a few resources where we have synthetic arns
+        # or they aren't in the iam ref docs.
+        allow_list = set((
```

Comment:
Perhaps `skip_list` rather than `allow_list`? Or `custodian_only_arns`.

Reply:
given its documented in detail on its usage and entries., this feels like a bike shed nit. I don't disagree but I don't see the value either.